src/components/VariantItem/VariantItem.js

```diff
@@ -63,10 +63,16 @@ class VariantItem extends Component {
 
     const variantPrice = priceByCurrencyCode(currencyCode, pricing);
 
+    const className = classNames(variantButton, {
+      [activeVariant]: isActive
+    }, {
+      [soldOutVariant]: variantInventoryStatus && variantInventoryStatus.type === "SOLD_OUT"
+    });
+
     return (
       <ButtonBase
         disableRipple
-        className={classNames(variantButton, { [activeVariant]: isActive }, { [soldOutVariant]: variantInventoryStatus && variantInventoryStatus.type === "SOLD_OUT" })}
```

Comment:
Is this change meant to be in this PR? Seems out of place with the rest of the changes.

Reply:
It's not related, but it was an eslint warning so I fixed it